### ISSUE TITLE
Incorrect computation of windows CPU core usage

### ISSUE THREAD

**Issue by Sebastian-0** [oshi/oshi#646]
Hi,

I am trying to use OSHI to implement a task manager of my own and when I started using the `getProcessorCpuLoadBetweenTicks()` function to measure the current CPU load for each core I realized that it is highly inaccurate. The higher the load you put the system under the more inaccurate it becomes, ending up with all the loads being 0.

After digging through the code, as well as the MSDN documentation, I realized that you have made a mistake when fetching the process usage. The process counter for "% Processor Time" actually returns the idle ticks if you use the raw values. The documentation says: "It is calculated by measuring the time that **the processor spends executing the thread of the Idle process** in each sample interval, and subtracting that value from 100%".

The code currently looks like this (WindowsCentralProcessor, lines 374-398):
```
// Fetch total processor ticks
// Later decrement by system + user
ticks[cpu][TickType.IDLE.getIndex()] = PerfDataUtil.queryCounter(this.processorTickCounter[cpu]);

...

ticks[cpu][TickType.IDLE.getIndex()] -= ticks[cpu][TickType.SYSTEM.getIndex()] + ticks[cpu][TickType.USER.getIndex()];
```

The last line should be removed, currently it's subtracting the actual processing time from the idle time, resulting in negative values when the load is high. When I fixed this the function works as expected and perfectly mirrors what the windows task manager reports. Also keep in mind that the comments are incorrect.

I currently don't have time to make a pull-request for this myself but if I do later I will reference it here.

**Comment by dbwiddis** [comment]
Hmm... this has been a headache for some time.  I got different PDH counter values for idle time so I decided to use the (consistent) processor time.  Obviously I didn't read the docs well enough.   I'll research this and hopefully fix it as you describe (unless I see some better way) but hope you can test the fix once it's in.

**Comment by Sebastian-0** [comment]
Sure, no problem. I think the documentation over at MSDN is poor for this use case, it took quite some time for me to figure out how it is supposed to work. While you do your research I will just recompile your library myself to make it work for my program.

**Comment by dbwiddis** [comment]
Research already done.  The key factor was noting that the counter type was [PERF_100NSEC_TIMER_INV](https://msdn.microsoft.com/en-us/library/ms803963.aspx) which counts down as you described.  I'd still like to do some testing on both Win10 and Win7, so give me a day or two to get this fixed.

**Comment by dbwiddis** [comment]
Fix is in #647 which is currently in SNAPSHOT.  If you can test it, I'll merge and cut a patch release.

**Comment by Sebastian-0** [comment]
I can confirm that this is indeed fixed in #647, the code looks good as well.

**Comment by dbwiddis** [comment]
Cool. It's merged. If you're okay with SNAPSHOT for now I'll see if anything else pops up this week, and release 3.9.1 this weekend.
